src/main/resources/story/Imdb/imdb-add-high-rating-movies-to-watchlist.story

```diff
@@ -0,0 +1,21 @@
+Description: Login into an existing user account. Go to the fantasy movies&shows and from the first page add 8+ rating movies&shows to the watchlist.
+
+Lifecycle:
+Examples:
+{transformer=FROM_LANDSCAPE}
+|email	 |wry74954@nezid.com|
+|password|wry74954@nezid.com|
+
+Scenario: Login in.
+When I login to IMDB with <email> and <password>
+
+Scenario: Go to the fantasy movies&shows and from the first page add 8+ rating movies&shows to the watchlist.
+When I click on element located `By.xpath(//*[text()="Menu"])`
+When I click on all elements located `By.xpath(//*[@role="menuitem"]//*[text()="Browse Movies by Genre"])`
```

Comment:
Why are you clicking "all" the elements here?

Reply:
Probably I made a mistake with autocomplete Ctrl+Space. I changed it to clicking just on the element.